src/pydvl/value/sampler.py

```diff
@@ -44,6 +44,9 @@
       Frank, and Geoffrey Holmes. [Sampling Permutations for Shapley Value
       Estimation](http://jmlr.org/papers/v23/21-0439.html). Journal of Machine
       Learning Research 23, no. 43 (2022): 1–46.
+[^2]: <a name="wang2023data"></a>Wang, J.T. and Jia, R., 2023.
```

Comment:
This citekey seems to be wrong. It should be `wang_2023_data`. If it's not like that in the bibtex file in `docs/assets/pydvl.bib`, change it there too.

---

We use Zotero for bibliography and betterbiblatex to export with the citekey format
```
[auth:lower:postfix,_][veryshorttitle:skipwords:lower:postfix,_][year]
```
Alas this is the convention from the tfl website and is not detailed [in the docs](https://github.com/aai-institute/pyDVL/blob/develop/CONTRIBUTING.md#using-bibliography). For this PR however you only need to change the id in the key that's already present in that file